content/en/events/2023/kcscn/schedule.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Schedule
+type: docs
+weight: 3
+---
+
+**NOTE:** You **MUST** pick up your badge before attending the summit event.
+
+## CFP
+
+If you would like to submit a formal session, please use the [CFP Form](TODO).
+This form will be open from Monday, August 14th, and covers presentation, and collaboration style sessions.
+
+## Unconference
+
+Unconference sessions should use the GitHub issue <https://github.com/kubernetes/community/issues/TODO>.
+The unconference session will start after several talks. If you have a topic in mind, please add it to the Github Issue.
```

Comment:
Can swap these out for 'CFP & unconference coming soon' just so that people won't click on them and get confused when they don't work.^^;;

Reply:
Done. Thanks!